synapseclient/__main__.py

```diff
@@ -513,18 +513,24 @@ def _replace_existing_config(path, auth_section):
     with open(path, "r") as config_o:
         config_text = config_o.read()
 
+    section_name = (
+        "default" if profile_name == "default" else f"profile {profile_name}"
```

Comment:
NIT: No need to hard-code "default"
```suggestion
        profile_name if profile_name == "default" else f"profile {profile_name}"
```